config/samples/prometheus.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: monitoring.coreos.com/v1
```

Comment:
What is this used for?

Reply:
Not sure, this is adapted from: https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/user-guides/getting-started.md#deploying-prometheus